custom/covid/management/commands/update_case_index_relationship.py

```diff
@@ -0,0 +1,68 @@
+from xml.etree import cElementTree as ElementTree
+
+from django.core.management.base import BaseCommand
+
+from casexml.apps.case.mock import CaseBlock
+from dimagi.utils.chunked import chunked
+
+from corehq.apps.linked_domain.dbaccessors import get_linked_domains
+from corehq.apps.hqcase.utils import submit_case_blocks
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+
+
+BATCH_SIZE = 100
+CASE_TYPE = "lab_result"
+DEVICE_ID = __name__ + ".update_case_index_relationship"
+
+
+def needs_update(case):
+    if len(case.indices) != 1:
+        return False
+    index = case.indices[0]
+    return index.referenced_type == "patient" and index.relationship == "child"
+
+
+def case_block(case):
+    index = case.indices[0]
+    return ElementTree.tostring(CaseBlock.deprecated_init(
+        create=False,
+        case_id=case.case_id,
+        owner_id='-',
+        index={index.identifier: (index.referenced_type, index.referenced_id, "extension")},
+    ).as_xml()).decode('utf-8')
+
+
+def update_cases(domain):
+    accessor = CaseAccessors(domain)
+    case_ids = CaseAccessors(domain).get_case_ids_in_domain(CASE_TYPE)
+    print(f"Found {len(case_ids)} {CASE_TYPE} cases in {domain}")
+
+    case_blocks = []
+    for case_id in case_ids:
+        case = accessor.get_case(case_id)
```

Comment:
not sure how many cases we expect, but this would likely benefit from chunking the fetches as well. I think we have a case accessor that does that.

Reply:
Agreed, but I looked through `CaseAccessorSQL` and didn't find anything.

Reply:
`accessor.iter_cases`: https://github.com/dimagi/commcare-hq/blob/08eec95e4246bf8eea78805a16a969b5a905f4d4/corehq/form_processor/interfaces/dbaccessors.py#L414

Reply:
sweet, thank you